fecgov/fec-cms
# Update package-lock file with the latest dependencies

## Summary (required)

- Resolves #4325 

## Reviewers**
-  One approval requested

_Include a summary of proposed changes._

-   Installed the node packages and rebuild the package-lock.json file

## How to test:

- Checkout `develop` branch.  Pull the latest changes and  run `snyk test --file=package-lock.json` . Notice the Prototype pollution vulnerability shows up in the terminal as both HIGH and MEDIUM priority.
<img width="1228" alt="Screen Shot 2021-01-15 at 1 27 22 PM" src="https://user-images.githubusercontent.com/11650355/104764527-83242780-5735-11eb-8e5f-3c00524617b6.png">

- Checkout `feature/4325-update-package-lock`
- Delete node modules folder:  run `rm -rf node-modules/`
- Delete package-lock.json file: run `rm -rf package-lock.json`
- Install node packages: run `npm i ` and `npm run build`
- run `snyk test --file=package-lock.json` In the terminal,  prototype pollution vulnerability no longer appears (unlike the develop branch, screenshot attached above)


____
